ios/Sources/Component/InlineTitleNavigationBarScrollView.swift

```diff
@@ -0,0 +1,58 @@
+import SwiftUI
+import Introspect
+
+private struct ContentOffsetPreferenceKey: PreferenceKey {
+    static var defaultValue: CGFloat = 0
+
+    static func reduce(value: inout CGFloat, nextValue: () -> CGFloat) {}
+}
+
+/**
+ A vertical scroll view applies scrollEdgeAppearance when its content reaches the matching edge of the navigation bar.
+
+ This behavior is similar to the default behavior from iOS 15 SDK, so can be removed after migrating to Xcode 13.
+ */
+public struct InlineTitleNavigationBarScrollView<Content: View>: View {
+
+    public var content: Content
+    public var showsIndicators: Bool
+    @State private var shouldApplyScrollEdgeAppearance = true
+
+    public init(
+        showsIndicators: Bool = true,
+        @ViewBuilder content: () -> Content
+    ) {
+        self.showsIndicators = showsIndicators
+        self.content = content()
+    }
+
+    public var body: some View {
+        let _ = shouldApplyScrollEdgeAppearance
```

Comment:
What is this property?
It seems not used 👀 

There are some TODOs?

Reply:
This line is required to aware for the change of `shouldApplyScrollEdgeAppearance`.